.github/scripts/install_prereqs.sh

```diff
@@ -195,15 +203,15 @@ if [[ $1 == "release" ]]; then
   echo "DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}"
   echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
   # For Debugging
-  git clone https://github.com/GenomicsDB/GenomicsDB.git -b develop $NATIVE_BUILD_DIR
+  git clone https://github.com/GenomicsDB/GenomicsDB.git -b ng_nanoarrow $NATIVE_BUILD_DIR
```

Comment:
Will revert to develop once https://github.com/GenomicsDB/GenomicsDB/pull/338 is approved.